fleetspeak/src/e2etesting/setup/setup_components.go

```diff
@@ -280,7 +283,7 @@ func modifyFleetspeakClientConfig(configDir string, httpsListenAddress string, n
 // BuildConfigurations builds Fleetspeak configuration files for provided servers and
 // number of clients that are supposed to be started on different machines
 func BuildConfigurations(configDir string, serverHosts []string, numClients int, mysqlCredentials MysqlCredentials) error {
-	err := buildBaseConfiguration(configDir, mysqlCredentials, serverHosts)
+	err := buildBaseConfiguration(configDir, mysqlCredentials, "10.132.0.12:6060")
```

Comment:
This line and the whole `BuildConfigurations()` function should be updated after #273